### ISSUE TITLE
Inconsistent documentation for condition syntax

### ISSUE THREAD

**Issue by 153957** [PyTables/PyTables#473]
The [condition_syntax](https://github.com/PyTables/PyTables/blob/develop/doc/source/usersguide/condition_syntax.rst) is inconsistent.

Specifically in the section about functions in conditions:
```
sqrt(float|complex): float|complex - square root.
{abs}(float|complex): float|complex - absolute value.
```

Sometimes the function names have brackets around them, and sometimes they do not. I guess the brackets should indicate the choice between multiple function names. However, `{abs}` is alone, so the brackets should probably be removed around `abs`.

**Comment by andreabedini** [comment]
right, thanks for noticing

**Comment by andreabedini** [comment]
fixed in 5eaef686102610439a08069cdd6be3f2211b30a4
